armi/utils/plotting.py

```diff
@@ -956,6 +958,8 @@ def __init__(
             self.peak = peak
             self.avgHistogram = None
             self.eHistogram = None
+            self.peakHistogram = None
+            self.E = None
```

Comment:
These changes seem random, not sure how you caught that these instance variables were required here.

Obviously nobody has used this plotter in a while...

Reply:
Oh, I just ran `pylint` on the two files I was editing in this PR.  Based on how the code runs, these are not bugs, but it is extremely bad programming practice to define `self.` variables on-the-fly in the code, and not in the `__init__()` method.